python-package/lightgbm/sklearn.py

```diff
@@ -189,7 +189,11 @@ def __init__(self, func: _LGBM_ScikitCustomObjectiveFunction):
         """
         self.func = func
 
-    def __call__(self, preds: np.ndarray, dataset: Dataset) -> Tuple[np.ndarray, np.ndarray]:
+    def __call__(
+        self,
+        preds: np.ndarray,
+        dataset: Dataset,
+    ) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
I accidentally made this change manually, while trying to revert `ruff`'s change to the other `__call__()` call a bit further down... but I think we should keep this change. That accident just revealed that these were inconsistent with each other, and I like the one `{argument}: {type}` per line style.